### ISSUE TITLE
Do you support Skype and Skype for Business?

### ISSUE THREAD

**Issue by hsetty** [bdkjones/MuzzleApp#32]
I regularly use Skype and it does not seem to support it. Wanted to put in a good word to add this feature request.

**Comment by bdkjones** [comment]
Skype currently uses an old approach to screen sharing: snap 30 screenshots per second. This means Muzzle can’t detect it. I’m hoping the upcoming release will fix that by using Apple’s official APIs

**Comment by hsetty** [comment]
ok got it.. how about skype for business which was formerly Lync but looks
like is using much newer technology in the current version?

**Comment by keirthomasbryant** [comment]
Adding a vote for Skype for Business support (aka Lync). Corporate drones like me use this worldwide without any choice because the business has signed up to Microsoft Office 365. Using other screensharing software is forbidden. 

Coming up for a year soon since this request was made -- has there been any progress? 

(Tested just now and sadly DND was not activated. I have a dual-screen setup though which again in common in corporate environments -- might this've cause issues?)

**Comment by bdkjones** [comment]
Skype for Business doesn’t use the correct/modern screensharing API, so Muzzle can’t detect it. Older software uses a “take 40 screenshots every second and stitch them together into video” approach, which is why CPU use is much higher for those apps.

**Comment by keirthomasbryant** [comment]
Yeah I see the issue -- looking at Activity Monitor and top there appears to be no process you can even monitor for signs of screen sharing. It's just a nebulous Skype for Business executable. 

There's talks of Microsoft switching all this functionality over to Teams, I believe. That would make Skype for Business even worse than it is now, and it's pretty appalling, but they might switch around APIs at that point.

**Comment by karstenBriksoft** [comment]
are there any news on this? this thread seems to be 3 years old and Skype still isn't properly supported...

**Comment by bdkjones** [comment]
https://github.com/bdkjones/MuzzleApp/issues/32#issuecomment-348223604
